homeassistant/components/goalzero/sensor.py

```diff
@@ -1,15 +1,134 @@
 """Support for Goal Zero Yeti Sensors."""
 from __future__ import annotations
 
-from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
+    SensorEntity,
+    SensorEntityDescription,
+)
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_NAME
+from homeassistant.const import (
+    CONF_NAME,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_CURRENT,
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_POWER,
+    DEVICE_CLASS_SIGNAL_STRENGTH,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_VOLTAGE,
+    ELECTRIC_CURRENT_AMPERE,
+    ELECTRIC_POTENTIAL_VOLT,
+    ENERGY_WATT_HOUR,
+    PERCENTAGE,
+    POWER_WATT,
+    SIGNAL_STRENGTH_DECIBELS,
+    TEMP_CELSIUS,
+    TIME_MINUTES,
+    TIME_SECONDS,
+)
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from . import Yeti, YetiEntity
-from .const import DATA_KEY_API, DATA_KEY_COORDINATOR, DOMAIN, SENSOR_TYPES
+from .const import DATA_KEY_API, DATA_KEY_COORDINATOR, DOMAIN
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="wattsIn",
+        name="Watts In",
+        device_class=DEVICE_CLASS_POWER,
+        unit_of_measurement=POWER_WATT,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="ampsIn",
+        name="Amps In",
+        device_class=DEVICE_CLASS_CURRENT,
+        unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
+    SensorEntityDescription(
+        key="wattsOut",
+        name="Watts Out",
+        device_class=DEVICE_CLASS_POWER,
+        unit_of_measurement=POWER_WATT,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="ampsOut",
+        name="Amps Out",
+        device_class=DEVICE_CLASS_CURRENT,
+        unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
+    SensorEntityDescription(
+        key="whOut",
+        name="WH Out",
+        device_class=DEVICE_CLASS_ENERGY,
+        unit_of_measurement=ENERGY_WATT_HOUR,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
+    ),
+    SensorEntityDescription(
+        key="whStored",
+        name="WH Stored",
+        device_class=DEVICE_CLASS_ENERGY,
+        unit_of_measurement=ENERGY_WATT_HOUR,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Is it correct that this is not `STATE_CLASS_TOTAL_INCREASING`?

Reply:
This is a realtime reading for how many wh are in the battery. I'm guessing it could use `STATE_CLASS_TOTAL`?

Reply:
Ok. We've removed `STATE_CLASS_TOTAL`.